### PR TITLE
[UI] Home Page View

### PR DESCRIPTION
**Pull request by AliceHuang1027**
1. changed server.js for different presents for different log status

### CODE REVIEW

File: src/routes/renderRoutes.js [hunk 1]
```diff
@@ -14,5 +14,23 @@ routes.postgres = async (req, res) => {
     dbPassword: userAccount.dbPassword,
   });
 };
+routes.landingpage = async (req, res) => {
+  if(!req.session.username){
```
Comment (greenc123) [inline src/routes/renderRoutes.js:18]: We need to migrate away from usernames to only ids.

Comment (greenc123) [reply src/routes/renderRoutes.js:18]: `if (!req.session.id) {`

File: tests/integration/__snapshots__/welcome.test.js.snap [hunk 2]
```diff
@@ -114,97 +114,14 @@ exports[`test welcome page should render databases page correctly 1`] = `
...
...licefanz/Desktop/js/databases/node_modules/express/lib/router/layer.js:95:5)<br> &nbsp; &nbsp;at trim_prefix (/Users/alicefanz/Desktop/js/databases/node_modules/express/lib/router/index.js:317:13)<br> &nbsp; &nbsp;at /Users/alicefanz/Desktop/js/databases/node_modules/express/lib/router/index.js:284:7<br> &nbsp; &nbsp;at Function.process_params (/Users/alicefanz/Desktop/js/databases/node_modules/express/lib/router/index.js:335:12)<br> &nbsp; &nbsp;at next (/Users/alicefanz/Desktop/js/databases/node_modules/express/lib/router/index.js:275:10)<br> &nbsp; &nbsp;at SendStream.error (/Users/alicefanz/Desktop/js/databases/node_modules/serve-static/index.js:121:7)<br> &nbsp; &nbsp;at SendStream.emit (events.js:198:13)<br> &nbsp; &nbsp;at SendStream.error (/Users/alicefanz/Desktop/js/databases/node_modules/send/index.js:270:17)<br> &nbsp; &nbsp;at SendStream.onStatError (/Users/alicefanz/Desktop/js/databases/node_modules/send/index.js:421:12)<br> &nbsp; &nbsp;at next (/Users/alicefanz/Desktop/js/databases/node_modules/send/index.js:763:28)<br> &nbsp; &nbsp;at /Users/alicefanz/Desktop/js/databases/node_modules/send/index.js:771:23<br> &nbsp; &nbsp;at FSReqWrap.oncomplete (fs.js:153:21)</pre>
```
Comment (songz) [inline tests/integration/__snapshots__/welcome.test.js.snap:123]: y is the snapshot giving error?

**Comment by greenc123** [comment]
Completed this PR here: https://github.com/garageScript/databases/pull/157

**Pull request closed without merging**